components/service/fretboard/src/test/java/mozilla/components/service/fretboard/source/kinto/HttpURLConnectionHttpClientTest.kt

```diff
@@ -27,14 +30,46 @@ class HttpURLConnectionHttpClientTest {
         }
     }
 
+    @Test(expected = ExperimentDownloadException::class)
+    fun testNonHTTPURL() {
+        HttpURLConnectionHttpClient().get(URL("file://filename.json"))
+    }
+
+    @Test(expected = ExperimentDownloadException::class)
+    fun testOpenConnectionException() {
+        val url = mock(URL::class.java)
+        `when`(url.openConnection()).thenThrow(IOException())
+        HttpURLConnectionHttpClient().get(url)
+    }
+
     @Test(expected = ExperimentDownloadException::class)
     fun testGET404() {
-        testGETError(404)
+        testGETResponseCode(404)
     }
 
     @Test(expected = ExperimentDownloadException::class)
     fun testGET500() {
-        testGETError(500)
+        testGETResponseCode(500)
+    }
+
+    @Test(expected = ExperimentDownloadException::class)
+    fun testGET199() {
+        testGETResponseCode(199)
+    }
+
+    @Test(expected = ExperimentDownloadException::class)
+    fun testGET300() {
+        testGETResponseCode(300)
+    }
+
+    @Test
+    fun testGET200() {
+        testGETResponseCode(200)
+    }
+
+    @Test
+    fun testGET299() {
+        testGETResponseCode(299)
```

Comment:
`testGET200` and `testGET299` do not perform any assertions or verifications.

Reply:
Now it's corrected (when I wrote them I thought about the test passing if the method didn't throw any exceptions), now they have assertions. 

Turns out `testGET200` was redundant since `sinceTestGETWithoutHeaders` was testing the same situation, and I changed `testGET299` to perform assertions.